### PR TITLE
[BD-05]: Add missing occurrences of pluggable editor

### PR DESCRIPTION
**Pull request by giovannicimolin**
**TL;DR -** This implement a few missing pluggable editor methods in the `grades` and `leaderboard` sections.

JIRA: [BLENDED-783](https://openedx.atlassian.net/browse/BLENDED-783)

**What changed?**

- Pluggable submission editor is now loaded properly on the `Your grade` and `Top responses` sections

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [x] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

1. Checkout this branch.
2. Add a ora problem with self assessment only.
3. Enable the leaderboard (`top responses`).
4. Submit the problem and assess the response.
5. Check that the `your grade` section displays a correctly formatted reponse (no html code).
6. Check that the `top resposes` properly displays the response

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @giovannicimolin! I've created [BLENDED-783](https://openedx.atlassian.net/browse/BLENDED-783) to keep track of it in Jira.   More details are on the [BD-05](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529970851/BD-05+ORA+v3+Usability+Moderation) project page.


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

**Review by nsprenkle — APPROVED** [review]
From testing locally, seems to fix the 2 issue locations from the bug report ("Your Grade" and "Top Responses"). :shipit:

**Review by jnlapierre — APPROVED** [review]
LGTM (Code reviewed)
